packages/core/test/dialog/dialogTests.tsx

```diff
@@ -64,6 +64,69 @@ describe("<Dialog>", () => {
         document.body.removeChild(container);
     });
 
+    describe("accessibility support", () => {
+        const mountDefaults = (className: string) => {
+            const container = document.createElement("div");
+            document.body.appendChild(container);
```

Comment:
I think these could be in a `beforeEach` block. and `container` can live as a `let container: HTMLElement;` outside that block.